src/gwt/src/org/rstudio/core/client/command/AppCommand.java

```diff
@@ -107,12 +107,10 @@ public void onVisibleChanged(AppCommand command)
    
    public AppCommand()
    {
-      if (Desktop.isDesktopReady())
+      if (Desktop.isDesktop())
```

Comment:
Originally guarded these with `isDesktopReady` because DesktopMenuCallback is not available on boot and I was getting script errors when we tried to invoke it. As long as you aren't seeing those anymore this is fine! (And if you are I think it suggests we should just move the ready check inside the callback)

Reply:
I'm not seeing these anymore, so I think we should be good to go!